### PR TITLE
chore: Remove eth-optimism package

### PR DESCRIPTION
**Pull request by alter-eggo**
This pr removes eth-optimism package as it seems that there is no need in it. Is it duplicated in @across-protocol package

This change will increase speed of app downloading as it is resulted in 35% decreasing of vendors chunk:
`2.5Mb (gzipped) => 1.6Mb (gzipped)`

Screenshot 2022-10-27 at 23 15 25

Screenshot 2022-10-27 at 23 21 01

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -16,17 +16,18 @@
     arb-bridge-eth "^0.7.4"
     arb-bridge-peripherals "^1.0.5"
 
-"@across-protocol/contracts-v2@^0.0.50":
```
Comment (nicholaspai) [inline yarn.lock:19]: This does seem outdated let me add a PR to bump this version

File: yarn.lock [hunk 2]
```diff
@@ -2375,6 +2376,15 @@
     debug "^3.1.0"
     lodash.once "^4.1.1"
 
+"@dabh/diagnostics@^2.0.2":
```
Comment (nicholaspai) [inline yarn.lock:2379]: What is this package?

Comment (alter-eggo) [reply yarn.lock:2379]: hmm, I am not sure, I just ran command `yarn remove @eth-optimism`

**Review by nicholaspai — COMMENTED** [review]
This looks great thanks for this! I've added a PR that will commit a badly needed `yarn.lock` update so your code diff looks smaller: https://github.com/across-protocol/frontend-v2/pull/401

File: yarn.lock (hunk 2 above)
Comment (nicholaspai) [reply yarn.lock:2379]: weird

Comment (nicholaspai) [reply yarn.lock:2379]: agreed

**Comment by nicholaspai** [comment]
@alter-eggo hey can you rebase off master? Code diff should go down a lot

**Comment by alter-eggo** [comment]
> @alter-eggo hey can you rebase off master? Code diff should go down a lot

done

File: yarn.lock [hunk 3]
```diff
@@ -8662,6 +8662,11 @@
   resolved "https://registry.yarnpkg.com/@uma/contracts-node/-/contracts-node-0.3.18.tgz#40e053ff6bc66f7ae7ce6c5d203a3200cf9ed928"
   integrity sha512-ENFYjvVnyplKZF1G5YxOfpm0Ob1nOKtKcwX+TtfOiQGLNMVed8azV8tTSAWlJG/5SZkekX9Y5xZSP5YfcmMj0A==
 
+"@uma/contracts-node@^0.3.18":
```
Comment (nicholaspai) [inline yarn.lock:8665]: nit: i ran the above change in `package.json` locally and didn't see this addition to yarn.lock, any idea why?

**Review by nicholaspai — APPROVED** [review]
Left one comment

File: yarn.lock (hunk 3 above)
Comment (alter-eggo) [reply yarn.lock:8665]: same for me, it's rather weird. may be during merging smth went wrong

Comment (alter-eggo) [reply yarn.lock]: I removed it, as it is duplication of 8660-8664 lines

Comment (alter-eggo) [reply yarn.lock]: weird thing

Comment (nicholaspai) [reply yarn.lock]: got it

**Comment by alter-eggo** [comment]
great!

Screenshot 2022-10-28 at 01 42 45
Screenshot 2022-10-28 at 08 54 22

**Pull request merged**